Doc/library/argparse.rst

```diff
@@ -1792,6 +1792,9 @@ Sub-commands
      >>> parser.parse_args(['2', 'frobble'])
      Namespace(subparser_name='2', y='frobble')
 
+   .. versionadded:: 3.7
```

Comment:
```suggestion
   .. versionchanged:: 3.7
```

If I recall correctly, the version* directive apply to their parent object block (module, class, function, data), so this is more correct.  We don’t want `add_subparsers` to show up in a list of functions added in 3.7 :slightly_smiling_face:

Reply:
I just copied this syntax from https://docs.python.org/3/library/argparse.html#argparse.FileType, should I fix that too while I'm at it?

Reply:
https://www.sphinx-doc.org/en/1.5/markup/para.html#directive-versionadded seems inconclusive. They use `versionadded` and `versionchanged` for "new parameter" examples. I'll defer to your judgement on this one.